spec/core/request.rb

```diff
@@ -0,0 +1,30 @@
+require 'spec_helper'
+
+RSpec.describe "Request", :type => :request do
+  client = Caplinked::Client.new api_key: "API_KEY", api_host: 'sandbox.caplinked.com', api_scheme: 'https'
+
+  it "execute init" do
+    stub_request(:get, "https://sandbox.caplinked.com/api/v1/files/38187?workspace_id=5886").
```

Comment:
you dont need this stub here right bc you are not calling perform on the http request?